rest/RestGrid.js

```diff
@@ -23,15 +23,13 @@ export class RestGrid extends Component {
             {extraToolbarItems} = this.props,
             gridOptions = defaults(this.props.gridOptions, {onRowDoubleClicked: this.onRowDoubleClicked});
         return fragment(
-            vframe(
-                restGridToolbar({model, extraToolbarItems}),
-                frame(
```

Comment:
Not sure why we were wrapping this grid in a frame. Our grid component already renders it's contents within a frame (a div with a flex: auto property). I can't think of a reason why we'd want this double nesting of frames so I removed. Looks the same to me but wanted to leave a comment in case there was a purpose here that I missed.